meteor/client/ui/Settings/RundownLayoutEditor.tsx

```diff
@@ -1304,198 +334,142 @@ export default translateWithTracker<IProps, IState, ITrackedProps>((props: IProp
+								<td className="settings-studio-rundown-layouts-table__actions table-item-actions c3">
+									<button className="action-btn" onClick={(e) => this.downloadItem(item)}>
+										<FontAwesomeIcon icon={faDownload} />
 									</button>
-								</div>
-								{item.type === RundownLayoutType.DASHBOARD_LAYOUT ? (
-									<>
-										<div>{RundownLayoutsAPI.isDashboardLayout(item) ? this.renderActionButtons(item) : null}</div>
-										<div className="mod mls">
-											<button className="btn btn-primary right" onClick={() => this.finishEditItem(item)}>
-												<FontAwesomeIcon icon={faCheck} />
-											</button>
-											<button className="btn btn-secondary" onClick={() => this.onAddButton(item)}>
-												<FontAwesomeIcon icon={faPlus} />
-												&nbsp;
-												{t('Add button')}
-											</button>
-										</div>
-									</>
-								) : (
-									<>
-										<div className="mod mls">
-											<button className="btn btn-primary right" onClick={() => this.finishEditItem(item)}>
-												<FontAwesomeIcon icon={faCheck} />
-											</button>
+									<button className="action-btn" onClick={(e) => this.editItem(item)}>
+										<FontAwesomeIcon icon={faPencilAlt} />
+									</button>
+									<button className="action-btn" onClick={(e) => this.onDeleteLayout(e, item)}>
+										<FontAwesomeIcon icon={faTrash} />
+									</button>
+								</td>
+							</tr>
+							{this.isItemEdited(item) && (
+								<tr className="expando-details hl">
+									<td colSpan={4}>
+										<div>
+											<div className="mod mvs mhs">
+												<label className="field">
+													{t('Name')}
+													<EditAttribute
+														modifiedClassName="bghl"
+														attribute={'name'}
+														obj={item}
+														type="text"
+														collection={RundownLayouts}
+														className="input text-input input-l"
+													></EditAttribute>
+												</label>
+											</div>
+											<div className="mod mvs mhs">
+												<label className="field">
+													{t('Type')}
+													<EditAttribute
+														modifiedClassName="bghl"
+														attribute={'type'}
+														obj={item}
+														options={this.props.layoutTypes}
+														type="dropdown"
+														collection={RundownLayouts}
+														className="input text-input input-l"
+													></EditAttribute>
+												</label>
+											</div>
 										</div>
-									</>
-								)}
-							</td>
-						</tr>
-					)}
-				</React.Fragment>
-			))
+										<div>{this.renderElements(item, layout)}</div>
+										{layout?.supportedElements.length ? (
+											<div className="mod mls">
+												<button className="btn btn-secondary" onClick={(e) => this.onAddElement(item)}>
+													<FontAwesomeIcon icon={faPlus} />
+													&nbsp;
+													{t(`Add ${layout?.filtersTitle?.toLowerCase() ?? 'filter'}`)}
```

Comment:
This won't work the way you expect it to. *"filter"* will not be picked up by the string extractor and will not be translatable. Also, calling `.toLowerCase()` on a UI piece of string is a bad idea. German, for example, writes all nouns with capital letters. Strings mangled like that will always be broken.

Reply:
Very good point, fixed with https://github.com/olzzon/tv-automation-server-core/commit/da0f5d88f6335f75364697064f494192bf7a8738